src/main/update/LonghornUpdater.ts

```diff
@@ -50,8 +52,12 @@ export class MacLonghornUpdater extends MacUpdater {
     return await super.getUpdateInfoAndProvider();
   }
 
+  private readonly isArm64 = Electron.app.runningUnderRosettaTranslation || os.arch() === 'arm64';
+
   findAsset(assets: GithubReleaseAsset[]): GithubReleaseAsset | undefined {
-    return assets.find(asset => asset.name.endsWith('-mac.zip'));
+    const suffix = this.isArm64 ? '-mac.aarch64.zip' : '-mac.x86_64.zip';
```

Comment:
Does this mean that all the new files the upgrade-server delivers end with `-mac.ARCH.zip` ? Are users of older code going to see new versions they can upgrade to, if they're running code that still looks for `*-mac.zip` and none of the files match?  (Of course I don't know exactly what these assets are for...)

Reply:
Yes. No, Matt agreed to manually upload one version with both `-mac.zip` and `-mac.x86_64.zip` (with the same files) to support the upgrade.

Those are just the files you see in GitHub releases pages.